rig-bedrock/Cargo.toml

```diff
@@ -7,7 +7,7 @@ readme = "README.md"
 description = "AWS Bedrock model provider for Rig integration."
 
 [dependencies]
-rig-core = { version = "0.11.0", features = ["image"]  }
+rig-core = { version = "0.11.1", path = "../rig-core", features = ["image"]  }
```

Comment:
I'm not sure if this is the right way to do it, but I spot checked some other crates and this is what the pattern seems to be.

Reply:
I'm fairly certain this is correct as well!